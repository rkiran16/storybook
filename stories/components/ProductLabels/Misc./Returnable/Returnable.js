import { html } from "lit-html";

export const Returnable = ({ isReturnable = true }) => {
  if (isReturnable) {
    return html`
      <span class="position-relative me-2 d-flex align-items-center">
        <span class="m-2 material-icons">done</span>
        <strong class="sub-title-3 fw-bold">RETURNABLE</strong>
      </span>
    `;
  } else {
    return html`
      <span class="position-relative me-2 d-flex align-items-center">
        <span class="m-2 material-icons">clear</span>
        <strong class="sub-title-3 fw-bold">RETURNABLE</strong>
      </span>
    `;
  }
};
