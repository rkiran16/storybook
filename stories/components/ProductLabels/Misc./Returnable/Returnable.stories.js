import { Returnable } from "./Returnable";

export default {
  title: "Components/ProductLabels/Misc/Returnable",
  parameters: {
    layout: "centered",
    zeplinLink:
      "https://app.zeplin.io/project/62902a463701ffb0e532df2b/screen/63053ee9c9b39414a05fab9b",
  },
};

const ReturnableTemplate = (args) => Returnable(args);

export const Standard = ReturnableTemplate.bind({});
Standard.args = {
  isReturnable: true,
};
