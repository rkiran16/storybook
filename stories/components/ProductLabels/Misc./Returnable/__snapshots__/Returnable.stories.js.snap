// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/ProductLabels/Misc/Returnable Standard 1`] = `
Object {
  "_$litType$": 1,
  "strings": Array [
    "
      <span class=\\"position-relative me-2 d-flex align-items-center\\">
        <span class=\\"m-2 material-icons\\">done</span>
        <strong class=\\"sub-title-3 fw-bold\\">RETURNABLE</strong>
      </span>
    ",
  ],
  "values": Array [],
}
`;
